lib/screens/authenticate/sign_in.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:brew_crew/services/auth.dart';
+import 'package:flutter/material.dart';
+import 'package:brew_crew/shared/constans.dart';
+import 'package:brew_crew/shared/loading.dart';
+
+class SignIn extends StatefulWidget {
+  final Function toggleView;
+
```

Comment:
_

Reply:
taka sama sytułacja jak wcześniej, aby to rozwiązać moge po prostu w sumie użyć po prostu route'a